SPECS/rpm/rpm.spec

```diff
@@ -10,9 +10,13 @@ URL:            https://rpm.org
 Source0:        http://ftp.rpm.org/releases/%{name}-%(echo %{version} | cut -d'.' -f1-2).x/%{name}-%{version}.tar.bz2
 # The license for the files below is the same as for RPM as they have originally came from rpm.
 # The git repo is hosted by centos. The version below is centos 8 stable.
-Source3:        https://git.centos.org/rpms/python-rpm-generators/raw/c8s/f/SOURCES/python.attr
-Source4:        https://git.centos.org/rpms/python-rpm-generators/raw/c8s/f/SOURCES/pythondeps.sh
-Source5:        https://git.centos.org/rpms/python-rpm-generators/raw/c8s/f/SOURCES/pythondistdeps.py
+Source3:        https://src.fedoraproject.org/rpms/python-rpm-generators/raw/f40/f/python.attr
+#Source4:        https://git.centos.org/rpms/python-rpm-generators/raw/c8s/f/SOURCES/pythondeps.sh
+Source5:        https://src.fedoraproject.org/rpms/python-rpm-generators/raw/f40/f/pythondistdeps.py
+Source6:        https://src.fedoraproject.org/rpms/python-rpm-generators/raw/f40/f/pythondist.attr
+#Source7:        https://src.fedoraproject.org/rpms/python-rpm-generators/raw/f40/f/pythonname.attr
```

Comment:
this script doesn't work properly.. skipping it for now

Reply:
Resolved by changing lua import path